product-service/src/utils/apiResponseBuilder.ts

```diff
@@ -0,0 +1,41 @@
+interface responseInterface {
+    statusCode: number
+    headers: Object,
+    body: Object
+}
+
+const defaultHeaders = {
+    'Access-Control-Allow-Methods': '*',
+    'Access-Control-Allow-Headers': '*',
+    'Access-Control-Allow-Origin': '*'
+};
+
+const errorResponse = ( err: Error, statusCode: number = 500 ): responseInterface => {
+    console.log(
```

Comment:
why we are using `console.log` and `winston` together? We should use 1 approach in all files

Reply:
You right, i forgot about it)
Replaced on last commit